validator/rule_cache/dmhc_rule_cache/dmhc.cc

```diff
@@ -163,11 +163,11 @@ void dmhc_t::check_direct_k_hash_conflicts(meta_set_t *ops, bool *consider) {
           printf("%03x ",mtable_hashes[f][i]);
         printf(": [");
         for (int f=0;f<input_fields;f++)
-          printf("%" PRIu32 " ",ops[f].tags[0]);
+          printf("%" PRIu64 " ",ops[f].tags[0]);
```

Comment:
Shouldn't all of these be `PRIuPTR`?

Reply:
Technically yes, but using the host pointer size as the tag size is all a bit gross.  Maybe instead we should define a tag format printer set based on the architecture?

Reply:
Agreed. Perhaps define a `PRIuTAG` for use in these places